### PR TITLE
Setup snowflake for CICD pipeline

### PR DESCRIPTION
**Pull request by joshuadeanhall**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by rajdnp — APPROVED** [review]

File: release/snowflake/cicd.yml [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+                - !Sub "arn:aws:iam::${ProdAccountId}:root"
+            Action: 
+              - s3:Put*
+              - s3:Get*
+              - s3:List*
+            Resource: 
+              - !Sub "arn:aws:s3:::cep-${PrefixLower}-${Env}-${AWS::AccountId}-artifacts"
+              - !Sub "arn:aws:s3:::cep-${PrefixLower}-${Env}-${AWS::AccountId}-artifacts/*" 
+
+  SnowflakeDatabaseDatabaseBuildProjectRole:
```
Comment (ericzbeard) [inline release/snowflake/cicd.yml:85]: This role should reference the common build project policy.
```
ManagedPolicyArns:
        - Fn::ImportValue: !Sub "cep-${Env}-common-build-project-policy"
```

**Review by ericzbeard — CHANGES_REQUESTED** [review]

File: release/snowflake/cicd.yml [hunk 2]
```diff
@@ -0,0 +1,740 @@
...
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: codebuild.amazonaws.com
+        Version: '2012-10-17'
+
+  CommonBuildProjectPolicy:
```
[the remark this answers is not in the document]
Comment (joshuadeanhall) [reply release/snowflake/cicd.yml]: Removed this.

File: release/snowflake/cicd.yml (hunk 1 above)
Comment (joshuadeanhall) [reply release/snowflake/cicd.yml:71]: Modified this to match latest example

**Comment by joshuadeanhall** [comment]
Updated everything to match the most recent example.

**Review by ericzbeard — DISMISSED** [review]

**Review by ericzbeard — APPROVED** [review]

**Pull request merged**